taxi/views.py

```diff
@@ -44,6 +62,23 @@
     model = Car
 
 
+class CarCreateView(LoginRequiredMixin, generic.edit.CreateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:car-list")
```

Comment:
Code Style issue: Use `-` to split words in URL identification parameter `name`, not the `_`. So, it should be `car-list` instead of `car_list`.